src/main/java/mekanism/common/item/gear/ItemAtomicDisassembler.java

```diff
@@ -314,18 +349,40 @@ public DisassemblerMode byIndex(int index) {
             return MathUtils.getByIndexMod(MODES, index);
         }
 
+        // This is kinda hacky. A couple duplicates here to allow the different interfaces to play nicely.
```

Comment:
The `IHasTranslationKey` can just be removed it is for use in TextComponentUtils, IHasTextComponent is a more specific "implementation" of it (unless anything is specifically querying the translation key otherwise.